### PR TITLE
Sync api to another repo

### PR DESCRIPTION
**Pull request by ErikJiang**
Signed-off-by: bo.jiang <bo.jiang@daocloud.io>

**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
Expose kubean's API through a single repository.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: hack/sync_api.sh [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+###### Clean Up #######
+# clean_up(){
+#     rm -rf $TMP_DIR
+# }
+# trap clean_up
+
+# git clone.
+
+# This function installs a Go tools by 'go get' command.
```
Comment (Michelle951) [inline hack/sync_api.sh:44]:
```suggestion
# This function installs a Go tool by 'go get' command.
```

File: hack/sync_api.sh [hunk 2]
```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+# set -o xtrace
+
+# This script will sync kubean api declaration to single repo.
```
Comment (Michelle951) [inline hack/sync_api.sh:8]:
```suggestion
# This script will sync kubean api declaration to a single repo.
```

**Review by Michelle951 — COMMENTED** [review]
word suggestions for reference

File: hack/sync_api.sh (hunk 1 above)
Comment (ErikJiang) [reply hack/sync_api.sh]: done.

File: hack/sync_api.sh (hunk 2 above)
Comment (ErikJiang) [reply hack/sync_api.sh]: done.

**Pull request merged**